### PR TITLE
chore: jmeter job-executor cleanup

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
## This PR

* upgraded to newest jmeter version - see https://jmeter.apache.org/changes.html
* fixes some things in the jmeter example:
  * chmod +x entrypoint.sh is missing
  * jmeter/basiccheck.jmx is not available in stage `staging` (but job-executor tries to fetch it and fails)
  * although we are adding config.yaml in staging, we are executing jmeter test "basiccheck.jmx" - it should be "load.jmx"
* removes dynatrace extension (this example should really be a basic example)
* simplified wording a bit

### CODE REVIEW

File: jmeter/0.1.0/README.md [hunk 1]
```diff
@@ -4,36 +4,47 @@ This integration shows you how to run JMeter as a Kubernetes job using the [Jo
...
 ```docker
 FROM alpine:3.15
 ENV env=production
-ARG JMETER_VERSION="5.1.1"
+ARG JMETER_VERSION="5.4.2"
 ENV JMETER_HOME /opt/apache-jmeter-${JMETER_VERSION}
 ENV	JMETER_BIN	${JMETER_HOME}/bin
 ENV	JMETER_DOWNLOAD_URL  https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-${JMETER_VERSION}.tgz
 
-# Load additional extensions
```
Comment (oleg-nenashev) [inline jmeter/0.1.0/README.md:33]: Fine with me if this is not used in the example as it looks like

**Review by oleg-nenashev — APPROVED** [review]

File: jmeter/0.1.0/README.md (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply jmeter/0.1.0/README.md:33]: It's part of keptn/jmeter-service, but it's not needed in the "generic" example we are providing here.

**Comment by christian-kreuzberger-dtx** [comment]
Thanks for the approval. I'll leave this open for @TannerGabriel to take a look, waiting for his approval next week ;)

**Review by TannerGabriel — APPROVED** [review]
LGTM

**Pull request merged**